test/specs/helpers/formDataToJSON.spec.js

```diff
@@ -47,4 +47,25 @@ describe('formDataToJSON', function () {
       foo: ['1', '2']
     });
   });
+
+  it('should resist prototype pollution CVE', () => {
+    const formData = new FormData();
+
+    formData.append('foo[0]', '1');
+    formData.append('foo[1]', '2');
+    formData.append('__proto__.x', 'hack');
+    formData.append('constructor.prototype.y', 'value');
```

Comment:
@DigitalBrainJS I think I may have something that could be interesting for you in order to detect these problems before they got reported to you.

Here is an introduction to the pattern I was thinking of: https://fast-check.dev/blog/2023/09/21/detect-prototype-pollution-automatically/. If you think it can be useful, I can try to first plug it on formDataToJSON and then with your pointers extend its usage to other potentially vulnerable parts. (side-note: Im' the author of fast-check)